hook.sh

```diff
@@ -279,18 +279,25 @@ while true; do
 
   display_warnings
 
-  # Ask the question (not using "read -p" as it uses stderr not stdout)
-  echo -en "${BLUE}Proceed with commit? [e/y/n/?] ${NC}"
+  if [ -t 1 ] || [ ! -z ${FAKE_TTY+x} ]; then
```

Comment:
Sorry @ssbarnea, I should have been more specific. I know what `FAKE_TTY` is. I wasn't familiar with the `${VAR+x}` syntax, but I've found the answer here: http://stackoverflow.com/a/13864829